config/argocd/templates/0200-argocd.yaml

```diff
@@ -27,15 +27,6 @@ spec:
         cpu: 250m
         memory: 1Gi
     sharding: {}
-  dex:
-    openShiftOAuth: true
-    resources:
-      limits:
-        cpu: 500m
-        memory: 256Mi
-      requests:
-        cpu: 250m
-        memory: 128Mi
```

Comment:
This structure is deprecated. New definition later in the file, since it starts alphabetically later (`sso`) in the list of attributes.